### PR TITLE
[DF-7] All Photos Page

### PR DESCRIPTION
**Pull request by octobrekids**
This branch including by
- All photos page
- All photos page test
- All photos page storybook 
- All photos page layout

### CODE REVIEW

**Comment by yee2542** [comment]
I'd prefer `photo` the `allphotho` for this module name

**Comment by yee2542** [comment]
I'd prefer `photo` the `allphotho` for this module name

**Review by yee2542 — CHANGES_REQUESTED** [review]

File: src/common/components/Loader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Loading } from './styled';
+
+export const Loader = (): JSX.Element => {
+  return (
+    <Loading>
+      <div className="lds-ellipsis">
```
[the remark this answers is not in the document]
Comment (octobrekids) [reply src/common/components/Loader/index.tsx]: fix here d8d6e6c9deb64c5e68c963fb2b134a54b4cf419a

**Comment by yee2542** [comment]
WTF
- select not implment
- photoRuducer not implment
- too big on `/pages/allphotos/index.tsx`
- too fragment on `LightBox` component
- no test
- `#lightbox-img` should be a styledcomponent code base

**Comment by yee2542** [comment]
WTF
- select not implment
- photoRuducer not implment
- too big on `/pages/allphotos/index.tsx`
- too fragment on `LightBox` component
- no test
- `#lightbox-img` should be a styledcomponent code base
- too many unused components e.g. LoadedImage, GroupThumbnailPhoto
- pages `/upload` missing layout

**Comment by yee2542** [comment]
> WTF
> 
> * select not implment
> * photoRuducer not implment
> * too big on `/pages/allphotos/index.tsx`
> * too fragment on `LightBox` component
> * no test
> * `#lightbox-img` should be a styledcomponent code base
> * too many unused components e.g. LoadedImage, GroupThumbnailPhoto
> * pages `/upload` missing layout

**Pull request merged**